src/test/scala/cromwell/engine/backend/jes/JesBackendSpec.scala

```diff
@@ -1,13 +1,14 @@
 package cromwell.engine.backend.jes
 
+import java.net.URL
+
 import cromwell.binding.values.{WdlFile, WdlString}
 import cromwell.binding.{Call, CallInputs}
 import cromwell.engine.workflow.WorkflowOptions
-import org.scalatest.mock.MockitoSugar
 import org.scalatest.{FlatSpec, Matchers}
-import spray.json.JsObject
+import org.specs2.mock.Mockito
```

Comment:
Well if we shouldn't I can get around it, but I kinda like using `anyString`s etc.. for mocking instead of concrete dummy "abc" strings because IMO it makes it more obvious that we do not care about what the value is. And `MockitoSugar` doesn't have this which is why I mixed in `Mockito` instead.
But if we don't want specs2 dependency I can remove it.